packages/fcl/src/interaction-template-utils/utils/find-imports.js

```diff
@@ -0,0 +1,22 @@
+import {generateImport} from "./generate-import.js"
+
+export function findImports(cadence) {
+  let iports = []
+
+  let importsReg = /import \w+ from 0x\w+/g
+  let fileImports = cadence.match(importsReg) || []
+  for (const fileImport of fileImports) {
+    let importReg = /import (\w+) from (0x\w+)/g
+    let fileiport = importReg.exec(fileImport)
```

Comment:
If this var needs to be unique here can you use `matchFileImports` or something clearer

Reply:
Updated!